pkg/acceptance/helpers/role_client.go

```diff
@@ -114,6 +116,31 @@ func (c *RoleClient) GrantOwnershipOnAccountObject(t *testing.T, roleId sdk.Acco
 	require.NoError(t, err)
 }
 
+// TODO: move later to grants client
+func (c *RoleClient) RevokeCurrentGrantsFromSchemaObject(t *testing.T, roleId sdk.AccountObjectIdentifier, objectId sdk.SchemaObjectIdentifier, objectType sdk.ObjectType) {
```

Comment:
Maybe rename it to `MoveOwnershipAndRevokeCurrentGrantsFromSchemaObject`? It's a long name, but it mentions that it's calling `grant ownership` rather than `revoke privileges from`.

Reply:
Changed the signature to `func (c *RoleClient) GrantOwnershipOnSchemaObject(t *testing.T, roleId sdk.AccountObjectIdentifier, objectId sdk.SchemaObjectIdentifier, objectType sdk.ObjectType, outboundPrivileges sdk.OwnershipCurrentGrantsOutboundPrivileges)` instead